.github/actions/create-kind-cluster/action.yaml

```diff
@@ -0,0 +1,148 @@
+name: "Create a KinD cluster"
+description: |
+  Create a KinD cluster.
+inputs:
+  secure:
+    description: "Whether the KinD cluster should be created with a secure local registry configuration"
+    required: false
+    default: "false"
+  temp-cert-dir:
```

Comment:
suggestion: `cert-dir`. Saying "temp" doesn't really make sense as a parameter name. I think it's a translation from the original code.